src/docs/index.jsx

```diff
@@ -352,36 +369,54 @@ const Component = React.createClass({
 });
 
 const App = React.createClass({
+	contextTypes: {
+		router: object,
+	},
+
 	propTypes: {
 		children: any,
+		location: any,
+	},
+
+	goToPath(path) {
+		this.context.router.push(path);
 	},
 
 	renderCategoryLinks(items) {
 		if (_.isPlainObject(items)) {
+			const sortedItems = _.chain(items)
+				.toPairs()
+				.sortBy((pair) => pair[0])
+				.value();
+
 			return (
-				<ul>
-					{_.map(items, (kids, categoryName) => {
+				<VerticalListMenu selectedIndices={[]}>
+					{_.map(sortedItems, ([categoryName, kids]) => {
 						return (
-							<li key={categoryName}>
-								<b>{_.startCase(categoryName)}</b>
+							<Item hasExpander key={categoryName}>
+								<span>{_.startCase(categoryName)}</span>
 								{this.renderCategoryLinks(kids)}
-							</li>
+							</Item>
 						);
 					})}
-				</ul>
+				</VerticalListMenu>
 			);
 		}
 
 		return (
-			<ul>
-				{_.map(items, (componentName) => {
+			<VerticalListMenu selectedIndices={[]}>
+				{_.map(_.sortBy(items), (componentName) => {
 					return (
-						<li key={componentName}>
-							<Link to={`/components/${componentName}`}>{componentName}</Link>
-						</li>
+						<Item
+							key={componentName}
+							onSelect={_.partial(this.goToPath, `/components/${componentName}`)}
```

Comment:
Cause `Link`s render `<a>` tags and I want to navigate when the user selects an item in the `VerticalListMenu`